systemtests/src/main/java/com/github/streamshub/systemtests/constants/Labels.java

```diff
@@ -53,4 +53,12 @@ public static LabelSelector getKnpControllerLabelSelector(String clusterName) {
         return getKnpLabelSelector(clusterName, KafkaNamingUtils.controllerPoolName(clusterName), ProcessRoles.CONTROLLER);
     }
 
+    public static LabelSelector getKafkaPodLabelSelector(String clusterName) {
+        return new LabelSelectorBuilder().withMatchLabels(Map.of(
+                ResourceLabels.STRIMZI_CLUSTER_LABEL, clusterName,
+                ResourceLabels.STRIMZI_COMPONENT_TYPE_LABEL, Kafka.RESOURCE_KIND.toLowerCase(Locale.ENGLISH)
+            ))
+            .build();
+    }
```

Comment:
With this, you are getting all Kafka Pods, right?
Based on this, you don't want to check brokers and controllers separately, right?
If so, it will fail on timeout for rolling update when just broker/controller Pods were rolled.